tsconfig.json

```diff
@@ -5,7 +5,6 @@
     "outDir": "out",
     "lib": ["es2018"],
     "sourceMap": true,
-    "rootDir": "src",
```

Comment:
was removed because `await-semaphore` contains its own tsconfig, ref https://stackoverflow.com/questions/57422458/error-ts6059-file-is-not-under-rootdir-rootdir-is-expected-to-contain-al/65676678#65676678